Submitted By: Tushar Teredesai <tushar@linuxfromscratch.org>
Date: 2003-10-03
Initial Package Version: 1.4.1
Origin: Sent by Sourcemage Guru Hamish Greig
Description: Motif compilation fix. Needed only if optimization flags are used.

$LastChangedBy$
$Date$

diff -ru Build.orig/motif/lib/Xm/util/mkmsgcat.c Build/motif/lib/Xm/util/mkmsgcat.c
--- Build.orig/motif/lib/Xm/util/mkmsgcat.c	2002-09-06 17:06:06.000000000 +1000
+++ Build/motif/lib/Xm/util/mkmsgcat.c	2003-05-26 17:13:38.000000000 +1000
@@ -14,7 +14,7 @@
 #include <string.h>
 char buf[1024];
 
-char *OSF_COPYRIGHT = "$\n\
+char *OSF_COPYRIGHT = "\
 $  @OSF_COPYRIGHT@\n\
 $  COPYRIGHT NOTICE\n\
 $  Copyright (c) 1990, 1991, 1992, 1993 Open Software Foundation, Inc.\n\
@@ -28,7 +28,7 @@
 $ (c) Copyright 1996 Novell, Inc. \n\
 $ (c) Copyright 1996 FUJITSU LIMITED.\n\
 $ (c) Copyright 1996 Hitachi.\n\
-$";
+";
 
 char *OSF_COPYRIGHT_C = "/*\n\
  *\n\
